frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -690,7 +689,7 @@ function ReaderDictionary:onShowDictionaryLookup()
                     end,
                 },
                 {
-                    text = _("Search dictionary"),
+                    text = _("Look up"),
```

Comment:
`Lookup` ^^.

I'm half of a mind to actually keep it simple and just use `Search`, actually.

Reply:
No, `look up` the phrasal verb (as in `click look up to perform a lookup`).

> I'm half of a mind to actually keep it simple and just use Search, actually.

I'd prefer to just keep it `Search dictionary`.